### PR TITLE
PCM-3330 Make the invalid group error message more friendly

### PR DESCRIPTION
**Pull request by kunyan**

### CODE REVIEW

**Comment by kunyan** [comment]
@gandhikeyur @jtrantin pls review it

File: app/i18n/template.pot [hunk 1]
```diff
@@ -955,6 +955,10 @@ msgstr ""
 msgid "Product Version"
 msgstr ""
 
+#: app/cases/controller/detailsSection.js
```
Comment (jtrantin) [inline app/i18n/template.pot]: I think you don't need to touch these files, as they are generated somehow. @vrathee @gandhikeyur Can you confirm these changes are ok an will not break anything?

**Comment by renujhamtani** [comment]
@kunyan we dont touch the template.pot files. You can remove those changes from this PR. After that, I will merge this.

**Comment by renujhamtani** [comment]
looks good now.

**Pull request merged**